packages/app-content-pages/src/screens/About/About.js

```diff
@@ -100,7 +100,7 @@ function AboutPage() {
         />
         <MaxWidthContent>
           <StyledHeading
-            color={{ light: 'brand', dark: 'accent-1' }}
+            color={{ light: 'neutral-1', dark: 'accent-1' }}
```

Comment:
The change to `neutral-1` isn't reflected everywhere in Figma, but in components with `brand` teal against white (especially text), we need to increase the contrast. See https://github.com/zooniverse/front-end-monorepo/issues/5888